app/src/org/odk/collect/android/widgets/QuestionWidget.java

```diff
@@ -107,12 +114,15 @@ public void onClick(View v) {
 
         setOrientation(LinearLayout.VERTICAL);
         setGravity(Gravity.TOP);
-        setPadding(0, 7, 0, 0);
+        
+        //TODO: This whole view should probably be inflated somehow 
```

Comment:
I like the idea of adding your initials right after the TODO. This makes it easy for one to search for their TODOs so they can easily revisit them.